specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/machineLearningServices.json

```diff
@@ -4851,6 +4856,24 @@
       ],
       "type": "object"
     },
+    "ImageMetadata": {
+      "type": "object",
+      "description": "Specifies the Image Metadata about current ComputeInstance",
```

Comment:
"Returns metadata about the operating system image for this compute instance." < Let's use lower casing for compute instance in alignment with marketing/cela naming requirements.